io_scene_hubs/exporter.py

```diff
@@ -1,282 +0,0 @@
-    elif property_type == 'array':
-        return gather_array_property(export_settings, blender_object, target, property_name, property_definition)
-    else:
-        return __to_json_compatible(getattr(target, property_name))
-
-def gather_array_property(export_settings, blender_object, target, property_name, property_definition):
-    array_type = property_definition['arrayType']
-    type_definition = export_settings['hubs_config']['types'][array_type]
-    is_value_type = len(type_definition['properties']) == 1 and 'value' in type_definition['properties']
-    value = []
-
-    arr = getattr(target, property_name)
-
-    for item in arr:
-        if is_value_type:
-            item_value = gather_property(export_settings, blender_object, item, "value", type_definition['properties']['value'])
-        else:
-            item_value = gather_properties(export_settings, blender_object, item, type_definition)
-        value.append(item_value)
-    
-    return value
-
-def gather_material_property(export_settings, blender_object, target, property_name, property_definition):
-    blender_material = getattr(target, property_name)
-
-    if blender_material:
-        double_sided = not blender_material.use_backface_culling
-        material = gltf2_blender_gather_materials.gather_material(
-            blender_material, double_sided, export_settings)
-        return material
-    else:
-        return None
-
-def gather_collections_property(export_settings, blender_object, target, property_name, property_definition):
-    filtered_collection_names = []
-
-    collection_prefix_regex = None
-
-    if 'collectionPrefix' in property_definition:
-        collection_prefix = property_definition['collectionPrefix']
-        collection_prefix_regex = re.compile(r'^' + collection_prefix)
-
-    for collection in blender_object.users_collection:
-        if collection_prefix_regex and collection_prefix_regex.match(collection.name):
-            new_name = collection_prefix_regex.sub("", collection.name)
-            filtered_collection_names.append(new_name)
-        elif not collection_prefix_regex:
-            filtered_collection_names.append(collection.name)
-
-    return filtered_collection_names
-
-def export(blender_scene, selected, hubs_config, registered_hubs_components):
-    if bpy.data.filepath == '':
-        raise RuntimeError("Save project before exporting")
-
-    filepath = bpy.data.filepath.replace('.blend', '')
-    filename_ext = '.glb'
-
-    export_settings = {}
-    export_settings['timestamp'] = datetime.datetime.now()
-    export_settings['gltf_filepath'] = bpy.path.ensure_ext(filepath, filename_ext)
-
-    if os.path.exists(export_settings['gltf_filepath']):
-        os.remove(export_settings['gltf_filepath'])
-
-    export_settings['gltf_filedirectory'] = os.path.dirname(
-        export_settings['gltf_filepath']) + '/'
-    export_settings['gltf_format'] = 'GLB'
```

Comment:
So, we got rid of these settings, are there any that need to be set for things to work correctly? I remember setting some of these to non-standard values.

Reply:
I think we only had to configure these because we were manually calling export. These settings are all set via the checkboxes in the gltf export window.